featurebyte/service/info.py

```diff
@@ -1123,3 +1131,36 @@ async def get_batch_feature_table_info(
             created_at=batch_feature_table.created_at,
             updated_at=batch_feature_table.updated_at,
         )
+
+    async def get_static_source_table_info(
```

Comment:
still wondering - is it necessary to have this centralized info service, vs. having each API object's service implement it's own info method?

Reply:
I think it was centralized to make it easy to inject info from one asset into that for another asset (e.g. entity info in table info etc. We can discuss and refactor if necessary in a separate effort.

Reply:
i see - in that situation, i think it would still be better if the table service were the only one importing those dependencies, as that would make it clearer as to which objects actually depend on others.

but yea, no action needed for now!